website/docs/r/cloudwatch_event_bus_policy.html.markdown

```diff
@@ -0,0 +1,41 @@
+---
+subcategory: "EventBridge (CloudWatch Events)"
+layout: "aws"
+page_title: "AWS: aws_cloudwatch_event_bus_policy"
+description: |-
+  Provides a resource to create an EventBridge policy to support cross-account events.
+---
+
+# Resource: aws_cloudwatch_event_permission
+
+Provides a resource to create an EventBridge resource policy to support cross-account events.
+
+~> **Note:** EventBridge was formerly known as CloudWatch Events. The functionality is identical.
+
+~> **Note:** The cloudwatch eventbus policy resource is incompatible with the cloudwatch event permissions resource and will overwrite them.
+
+## Example Usage
+
+### Account Access
+
+```hcl
+resource "aws_cloudwatch_event_bus_policy" "test" {
+  policy         = data.aws_iam_policy_document.access.json
```

Comment:
Could you add this sample policy as part of the example to improve docs experience?

Reply:
docs added with multiple examples. LGTM